Makefile

```diff
@@ -2,12 +2,13 @@
 
 # programs
 MAKE=make
-AR=ar
+AR?=ar
```

Comment:
Again, why is this needed?   `AR=ar` can be overridden by `make AR=....` already.